calibration/FLORTD/CGINS-FLORTD-01197__20181206.csv

```diff
@@ -1,11 +1,11 @@
-serial,name,value,notes
-1197,CC_dark_counts_cdom,50,date in filename comes from dev file
-1197,CC_scale_factor_cdom,0.0652,
-1197,CC_dark_counts_chlorophyll_a,57,
-1197,CC_scale_factor_chlorophyll_a,.0114,
-1197,CC_dark_counts_volume_scatter,50,
-1197,CC_scale_factor_volume_scatter,2.072e-06,
-1197,CC_depolarization_ratio,0.039,Constant
-1197,CC_measurement_wavelength,700,[nm]; Constant
-1197,CC_scattering_angle,124,[degrees]; Constant
-1197,CC_angular_resolution,1.076,Erroneously named constant; this coefficient scales the particulate scattering at 124 degrees to total backscatter from particles
+serial,name,value,notes

```

Comment:
Mistakenly changed because new dev file had the wrong calibration date. Corrected and replaced original screw-up.